tests/functional/pages/surveys/census/household/overnight-visitors.page.js

```diff
@@ -11,6 +11,10 @@ class OvernightVisitorsPage extends QuestionPage {
     return browser.element('[name="overnight-visitors-answer"]').getValue()
   }
 
+  getErrorMsg() {
```

Comment:
If we re-generate the Selenium pages then this will probably get lost, is it something that is generic to all pages (could be in the base page model / added to a new base page to derive all these from?)